src/assets/icons/FacetLocalFindSearchIcon.svg

```diff
@@ -0,0 +1,4941 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 26.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
Can you add this in here https://github.com/CBIIT/datacommons-assets and use the URL

Reply:
@AjayDoddapaneni PR @ https://github.com/CBIIT/datacommons-assets/pull/17